views/users/delete.html

```diff
@@ -9,17 +9,17 @@
 {% endblock %}
 
 {% block main %}
-<h1>{{title}}</h1>
+<h1 class="base--h1">{{title}}</h1>
 {% if message %}
-<p class="form--alert" role="alert" for="delete">{{ message }}</p>
+<p class="base--p form--alert" role="alert" for="delete">{{ message }}</p>
```

Comment:
Nothing should have two classes on it. If you want the styles from `base--p` to carry over for `form--alert`, spin them out in to a mixin and have the mixin be called from both